hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/DUResponse.java

```diff
@@ -31,13 +31,16 @@
   private ResponseStatus status;
 
   /** The number of subpaths under the request path. */
-  @JsonProperty("count")
+  @JsonProperty("subPathCount")
   private int count;
 
   /** Encapsulates a DU instance for a subpath. */
-  @JsonProperty("data")
+  @JsonProperty("subPaths")
   private List<DiskUsage> duData;
 
+  @JsonProperty("keySize")
```

Comment:
```suggestion
  @JsonProperty("size")
```

This field's display name should be in accordance with DiskUsage's size field.